### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by savelichalex**

### CODE REVIEW

**Review by sertony — APPROVED** [review]

File: kit/controls/src/UIActionButton/hooks.ts [hunk 1]
```diff
@@ -28,9 +28,9 @@ export const useButtonColorScheme = (type: UIActionButtonType): ActionButtonColo
                         disabled: ColorVariants.BackgroundNeutral,
                     },
                     content: {
-                        normal: ColorVariants.TextPrimary,
-                        hover: ColorVariants.TextPrimary,
-                        pressed: ColorVariants.TextPrimary,
+                        normal: ColorVariants.TextPrimaryInverted,
+                        hover: ColorVariants.TextPrimaryInverted,
+                        pressed: ColorVariants.TextPrimaryInverted,
```
Comment (SAnatoliiS) [inline kit/controls/src/UIActionButton/hooks.ts]: Why `TextPrimaryInverted `? According to the layouts should be `TextPrimary `
image

Comment (savelichalex) [reply kit/controls/src/UIActionButton/hooks.ts]: It was black for light theme :)

Comment (savelichalex) [reply kit/controls/src/UIActionButton/hooks.ts]: Screenshot_2021-12-18_at_20 09 35

Comment (SAnatoliiS) [reply kit/controls/src/UIActionButton/hooks.ts]: Why do you think it's wrong? If so, then you need to ask to change the color on the layouts.

Comment (SAnatoliiS) [reply kit/controls/src/UIActionButton/hooks.ts]: It will be black with `TextPrimaryInverted` with dark theme instead

Comment (savelichalex) [reply kit/controls/src/UIActionButton/hooks.ts]: IDK, for me it's obvious that it's wrong 😂

Comment (savelichalex) [reply kit/controls/src/UIActionButton/hooks.ts]: changed)

**Review by sertony — APPROVED** [review]

File: kit/controls/src/UIActionButton/hooks.ts (hunk 1 above)
Comment (SAnatoliiS) [reply kit/controls/src/UIActionButton/hooks.ts]: `StaticTextPrimaryLight` is ok for me. Can you ask @eteslov to change the color on the layouts?

**Review by SAnatoliiS — APPROVED** [review]

**Pull request merged**